.github/workflows/fevm-contract-tests.yaml

```diff
@@ -0,0 +1,150 @@
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: Install Foundry
+        working-directory: ipc/contracts
+        run: |
+          curl -L https://foundry.paradigm.xyz | bash
+          export PATH="$PATH:/home/runner/.config/.foundry/bin"
+          foundryup
+      - name: Create a docker build
+        id: docker-build
+        working-directory: ipc/fendermint
+        run: |
+          export PATH="$PATH:/home/runner/.config/.foundry/bin"
+          make docker-build
+      - name: Run a testnode
+        id: testnode
+        working-directory: ipc/
+        run: |
+          export BALANCE=10000000000
+          { out=$(FM_PULL_SKIP=true cargo make --makefile ./infra/fendermint/Makefile.toml testnode | tee /dev/fd/3); } 3>&1
+          private_key=$(echo $out | sed -e 's/\(.*\)\([a-f0-9]\{64\}\)/0x\2/' | grep 0x | head -c66)
+          echo "ROOT_PRIVATE_KEY=$private_key" >> "$GITHUB_OUTPUT"
+          cat $GITHUB_OUTPUT
+      - name: 'Create and fund accounts'
+        id: accounts
+        env:
+          NUM_ACCOUNTS: 5
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+        working-directory: fevm-contract-tests
+        run: |
+          npx hardhat --network local create-fund-accounts
+      - name: 'Run tests: web3.js SimpleCoin'
+        if: always()
+        env:
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+        working-directory: fevm-contract-tests
+        run: |
+          npx hardhat --network local test ./test/web3.js/SimpleCoin.js
+      - name: 'Run tests: web3.js ERC20'
+        if: always()
+        timeout-minutes: 2
+        working-directory: fevm-contract-tests
+        env:
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+        run: |
+          npx hardhat --network local test ./test/web3.js/ERC20.js
+      - name: 'Run tests: ethers.js SimpleCoin'
+        if: always()
+        timeout-minutes: 2
+        working-directory: fevm-contract-tests
+        env:
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+        run: |
+          npx hardhat --network local test ./test/ethers.js/SimpleCoin.js
+      - name: 'Run tests: ethers.js ERC20'
+        if: always()
+        timeout-minutes: 2
+        working-directory: fevm-contract-tests
+        env:
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+        run: |
+          npx hardhat --network local test ./test/ethers.js/ERC20.js
+      - name: 'Run tests: Uniswap -- SKIPPING'
+        if: always()
```

Comment:
Why not something like `if: never()` and then CI would say `Skipped` on its own?

Reply:
I looked into that and if:never() is not a feature offered by github - I think it makes sense to remove the steps from this PR that are skipped and opened https://linear.app/interplanetary-consensus/issue/ENG-688/remove-skipped-tests-uniswap-and-openzeppelin-from-pr